widgetssdk/src/main/java/com/glia/widgets/chat/model/ChatState.java

```diff
@@ -484,5 +495,18 @@ public Builder setVisitorContextAssetId(String visitorContextAssetId) {
             this.visitorContextAssetId = visitorContextAssetId;
             return this;
         }
+
+        public Builder enableChatPanel() {
+            setChatInputMode(ChatInputMode.ENABLED);
```

Comment:
I see `setShowSendButton(false)` in `disableChatPanel()`.
May be we need to add `setShowSendButton(true)` to `enableChaaatPanel()` function accordingly?

Reply:
It seems send button needs to be shown after text input.